lib/resources/dstu2/document_reference.yaml

```diff
@@ -35,7 +35,20 @@ fields:
         }
       ]
     }
-  note: The type must include a loinc coding.
+
+              <b>(OR)</b>
+
+    {
+      "coding": [
+        {
+          "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/72",
+          "code": "3374547"
+        }
+      ]
+    }
```

Comment:
this is rendered on the site as one large code block

I would not write "(OR)", just provide both examples, ideally in two code blocks - may need to tweak the table generator

also, that's a very long URL, and causes the whole table to start scrolling

Reply:
Done: https://github.com/akhilbukka/fhir.cerner.com/commit/e1c33603617e33993b6e48bed7d6ac32e3b983ec#diff-eefc10b8b4a495f72e678cead5851448R39